src/routes/Dashboard/SummaryCard/index.jsx

```diff
@@ -1,27 +1,65 @@
+import {useState} from "react";
 import {BiTime, BiDollarCircle} from "react-icons/bi";
+import {DateTime} from "luxon";
 import Button from "components/main/Button";
 import SummaryCardSection from "routes/Dashboard/SummaryCard/components/SummaryCardSection";
 import CounterTime from "routes/Dashboard/SummaryCard/components/CounterTime";
+import DeleModal from "components/modals/DeleModal";
 
 export default function SummaryCard() {
+    const [isPrint, setIsPrint] = useState(false)
+    const [switchBtn, setSwitchBtn] = useState(false)
```

Comment:
same here.

`showPrintBtn` would be something better

Reply:
actually that's related more to the counter, so `counterIsActive` sounds like the best wording here